services/frontend-service/pkg/cmd/server.go

```diff
@@ -79,9 +81,14 @@ func RunServer() {
 		if err != nil {
 			logger.FromContext(ctx).Fatal("grpc.deployService.register", zap.Error(err))
 		}
+		err = api.RegisterBatchServiceHandlerServer(ctx, grpcProxy, &gproxy)
+		if err != nil {
+			logger.FromContext(ctx).Fatal("grpc.batchService.register", zap.Error(err))
+		}
 
 		mux := http.NewServeMux()
 		mux.Handle("/environments/", grpcProxy)
+		mux.Handle("/batches/", grpcProxy)
```

Comment:
You are registering the batch endpoint here as "/batches/" but in the proto file it's "/batches" ( https://github.com/freiheit-com/kuberpult/blob/main/pkg/api/api.proto#L38 ). Which one is this correct?

Reply:
hmm. Why didn't this break the code for me? 
I'll change it here